UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/DistributionService.cs

```diff
@@ -7,7 +8,16 @@ namespace UKHO.D365CallbackDistributorStub.API.Services
     public class DistributionService
     {
         private static readonly Queue<DistributorRequest> s_recordDistributorRequestQueue = new();
-        public static bool SaveDistributorRequest(CustomCloudEvent cloudEvent)
+        private static readonly List<CommandDistributionRequest> s_CommandDistributionList = new();
+        private const HttpStatusCode _ok = HttpStatusCode.OK;
```

Comment:
Private consts should be Pascal case please:
```suggestion
        private const HttpStatusCode Ok = HttpStatusCode.OK;
```